updatecli/policies/file/updatecli.d/default.yaml

```
@@ -0,0 +1,61 @@
+---
+name: Copy files {{ .sourceFiles }} from {{ requiredEnv .scm.source.owner}}/{{ .scm.source.repository}}
+
```

```suggestion
#{{ if .pipelineid }}
pipelineid: '{{ .pipelineid }}'
#{{ end }}
```

It would be nice to enforce a specific pipelineid in case we want to group this pullrequest with another one
for example https://github.com/updatecli/policies/blob/82a3f1771abaab9fcccfd1e49bf1ae1505483805/updatecli/policies/autodiscovery/cargo/updatecli.d/default.tpl#L9